EnglishFilter/sections/css_extended.txt

```diff
@@ -3,6 +3,9 @@
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/46405
 pornpictureshq.com#?#.masonry_thumbs > div.grid > div.thumb:has(> iframe)
+! https://github.com/AdguardTeam/AdguardFilters/issues/46561
+imdb.com###maindetails_sidebar_bottom > div.aux-content-widget-2:has(> span.ab_widget > div.ab_zergnet)
+imdb.com###sidebar > div.aux-content-widget-2:has(> span.ab_widget > div.ab_zergnet)
```

Comment:
Could you please change `##` to `#?#`, like in other extended CSS rules?

```
imdb.com#?##maindetails_sidebar_bottom > div.aux-content-widget-2:has(> span.ab_widget > div.ab_zergnet)
imdb.com#?##sidebar > div.aux-content-widget-2:has(> span.ab_widget > div.ab_zergnet)
```

Reply:
@AdamWr  done, thanks

Reply:
Thank you.